examples/gallery/plot/solar.py

```diff
@@ -0,0 +1,46 @@
+"""
```

Comment:
This gallery example could be moved into the "embellishment" category as per the comment https://github.com/GenericMappingTools/pygmt/issues/876#issuecomment-792923723

Reply:
Changed in 386ea48